### PR TITLE
multienv values

### PR DESCRIPTION
**Pull request by francardoso93**

### CODE REVIEW

File: helm/values-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  minReplicas: 1
+  maxReplicas: 5
+  targetCPUUtilizationPercentage: 70
+  targetMemoryUtilizationPercentage: 70
+env:
+  nodeEnv: dev
+  peerIdS3Bucket: us-west-2-dev-ep-bitswap-config
+  concurrency: 128
+  pipelining: 16
+  cacheBlocksInfo: false
```
Comment (blucas) [inline helm/values-dev.yaml]: Can't these duplicates be set in the base?

File: helm/values.yaml [hunk 2]
```diff
@@ -1,33 +1,8 @@
...
-    memory: 8Gi
-  requests:
-    cpu: 3
-    memory: 2Gi
-autoscaling:
-  enabled: true
-  minReplicas: 2
-  maxReplicas: 40
-  targetCPUUtilizationPercentage: 70
-  targetMemoryUtilizationPercentage: 70
```
Comment (blucas) [inline helm/values.yaml]: Do we want to set sane defaults that envs could just override?

**Review by blucas — CHANGES_REQUESTED** [review]

File: helm/values-dev.yaml (hunk 1 above)
Comment (francardoso93) [reply helm/values-dev.yaml]: Done

Comment (francardoso93) [reply helm/values-dev.yaml]: Done!

File: helm/values.yaml (hunk 2 above)
Comment (francardoso93) [reply helm/values.yaml]: Done!

**Review by blucas — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**